docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3.9"
+networks:
+  example_network:
+    driver: bridge
+
+services: 
+  mqtt:
+    image: eclipse-mosquitto
+    volumes:
+      - ./mosquitto/conf:/mosquitto/config
+      - ./mosquitto/data:/mosquitto/data
+      - ./mosquitto/log:/mosquitto/log
+    ports:
+      - 1883:1883
+      - 9001:9001
+    networks:
+      - example_network
```

Comment:
We dont need broker service here. I'm planning to create a different cocker-compose.yaml for benchmark scripts.